src/Microsoft.OpenApi.Readers/V2/OpenApiV2VersionService.cs

```diff
@@ -221,5 +221,11 @@ public OpenApiDocument LoadDocument(RootNode rootNode)
         {
             return (T)_loaders[typeof(T)](node);
         }
+
+        /// <inheritdoc />
+        public string GetReferenceScalarValues(MapNode mapNode, string scalarValue)
+        {
+            throw new NotImplementedException();
```

Comment:
not implemented or invalid operation?

Reply:
InvalidOperation..thanks for pointing this out